README.md

```diff
@@ -123,6 +120,23 @@ run of bootstrap:
 
 Both scripts can be used to do simple limited operation through the jcli debugging tools.
 
+## Further Build Commands
+
+Usually you'll use the master-branch version. To switch to a [specific release tag](https://github.com/input-output-hk/jormungandr/releases/latest)
+
+```sh
+git checkout tags/<specific-release-tag> #replace this with something like v1.2.3
+git submodule update  # always update the submodules after a version change
```

Comment:
This is required even when building off master.

Reply:
The `--recurse-submodules` does all the work.

If you mean the 2 lines: please see https://github.com/input-output-hk/jormungandr/pull/1541#discussion_r365914372